### PR TITLE
完善 logdb search service 类，允许设置搜索时间范围

### PR DESCRIPTION
**Pull request by andrewei1316**
## Changes
   
  - [ ] feature1
  - [ ] feature2 
  - [ ] fixbug1
  - [ ] fixbug2
 
## Reviewers

  - [ ] @wangtuo  please review
  - [ ] @guifeng please review
    
## Checklist
   
   - [ ] Rebased/mergable
   - [ ] Tests pass
   - [ ] CHANGELOG.md updated
   - [ ] Jira issue/task done

### CODE REVIEW

File: sdk/src/main/java/com/qiniu/pandora/logdb/SearchService.java [hunk 1]
```diff
@@ -21,8 +24,33 @@
...
+
     private SearchRequest sr = new SearchRequest();
 
+    private void setQueryWithDate() {
+        if (this.timeField == null || this.timeField.equals("") || this.startTime == 0) {
+            return;
+        }
+        String query = this.sr.getQuery();
+        if(endTime == 0) {
+            Calendar instance = Calendar.getInstance();
```
Comment (gf53520) [inline sdk/src/main/java/com/qiniu/pandora/logdb/SearchService.java]: System.currentTimeMillis()

**Comment by wangtuo** [comment]
lgtm

File: example/logdb/src/main/java/Search.java [hunk 2]
```diff
@@ -21,7 +21,7 @@ public static void main(String[] args) {
     try {
       SearchService searchService = logDBClient.NewSearchService();
       SearchService.SearchRet searchRet = searchService.setFrom(0).setSize(1).
-              setQuerystring("*").setRepo(repo).setHighlight(hl)
+              setQuerystring("*").setRepo(repo).setSort("field").setHighlight(hl)
```
Comment (longbai) [inline example/logdb/src/main/java/Search.java:24]: 不是变量？

Comment (andrewei1316) [reply example/logdb/src/main/java/Search.java:24]: 这个是一个 example ,不是业务代码

Comment (gf53520) [reply example/logdb/src/main/java/Search.java:24]: 这个是example

**Comment by gf53520** [comment]
LGTM

**Pull request merged**